jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/impl/util/DeadlineSchedulerHelper.java

```diff
@@ -138,7 +138,7 @@ public static void scheduleDeadlines(final List<? extends Deadline> deadlines, f
         for (Deadline deadline : deadlines) {
             if (Boolean.FALSE.equals(deadline.isEscalated())) {
                 Date date = deadline.getDate();
-                deadlineService.schedule(taskId, deadline.getId(), date.getTime() - now, type);
+                deadlineService.schedule(taskId, deadline.getId(), date.getTime() - now, type, true);
```

Comment:
This is also invoked from rescheduleDeadlinesForTask methods, where this should be false